administrator/language/en-GB/com_languages.ini

```
@@ -148,12 +150,14 @@ COM_LANGUAGES_VIEW_OVERRIDES_LANGUAGES_BOX_ITEM="%1$s - %2$s"
 COM_LANGUAGES_VIEW_OVERRIDES_N_ITEMS_DELETED="%d language overrides were deleted."
 COM_LANGUAGES_VIEW_OVERRIDES_N_ITEMS_DELETED_1="Language override was deleted."
 COM_LANGUAGES_VIEW_OVERRIDES_NO_ITEM_SELECTED="You haven't selected any overrides."
+COM_LANGUAGES_VIEW_OVERRIDES_NOT_TRANSLATED="Not translated"
```

```suggestion
COM_LANGUAGES_VIEW_OVERRIDES_NOT_TRANSLATED="Not Translated"
```

please follow the style guide